### PR TITLE
Add code to reverse array

### PR DESCRIPTION
**Pull request by Dorje64**
Added code to reverse array tdd

### CODE REVIEW

File: codes/reverser.rb [hunk 1]
```diff
@@ -3,5 +3,9 @@ class << self
     def reverse_string(string_to_reverse)
       string_to_reverse.reverse
     end
+
+    def reverse_array(array)
+    	return array.reverse
```
Comment (xhagrg) [inline codes/reverser.rb:8]: unnecessary return.

File: spec/tdd_example_spec.rb [hunk 2]
```diff
@@ -1,16 +1,16 @@
 require_relative '../codes/reverser.rb'
 
 describe 'passed values should be reversed' do
-  # let!(:test) { print 'hello world abc.    ' }
-  # let(:another_test) { print 'Hi! I am another test.   ' }
+  let!(:test) { print 'hello world abc.    ' }
```
Comment (xhagrg) [inline spec/tdd_example_spec.rb:4]: is this necessary?

Comment (asheshbyanju) [inline spec/tdd_example_spec.rb:4]: was that necessary dorj???

File: spec/tdd_example_spec.rb [hunk 3]
```diff
@@ -22,12 +22,12 @@
   describe '.reverse_array' do
     it 'reverses the array passed' do
```
Comment (xhagrg) [inline spec/tdd_example_spec.rb:23]: more test cases required.

File: spec/tdd_example_spec.rb (hunk 2 above)
Comment (ShradhaBasnet) [inline spec/tdd_example_spec.rb:4]: is that necessary??

**Pull request closed without merging**